packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -87,6 +94,7 @@ const Updates = decorate([
   addSubscriptions({
     backupNgJobs: subscribeBackupNgJobs,
     jobs: subscribeJobs,
+    proxyIds: cb => subscribeProxies(proxies => cb(map(proxies, 'id'))),
```

Comment:
I wonder if a one time call would be enough for this. It would lower the amount of subscriptions in xo-web.